pkg/handlers/app.go

```diff
@@ -20,6 +20,51 @@ import (
 	"github.com/replicatedhq/kots/pkg/version"
 )
 
+func (h *Handler) GetPendingApps(w http.ResponseWriter, r *http.Request) {
```

Comment:
this function seems to be returning 1 pending app, maybe the name should be `GetPendingApp` instead of `GetPendingApps`?